### PR TITLE
Sync web/guide/html/content_categories

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 1]
```diff
@@ -2,96 +2,98 @@
...
+## 主内容类
 
-### 元数据内容（Metadata content）
+### 元数据内容
 
-此类元素 修改文档其余部分的陈述或者行为，建立与其他文档的链接，或者传达其他外带信息。
+属于*元数据内容*（Metadata content）的元素可以修改文档其余部分的呈现或行为，建立与其他文档的链接，或者传达其他*外带*信息。
 
-属于这一类的元素有：{{HTMLElement("base")}}, {{Deprecated_Inline}}{{HTMLElement("command")}}, {{HTMLElement("link")}}, {{HTMLElement("meta")}}, {{HTMLElement("noscript")}}, {{HTMLElement("script")}}, {{HTMLElement("style")}} and {{HTMLElement("title")}}.
+属于这一类的元素有：{{HTMLElement("base")}}、{{Deprecated_Inline}}{{HTMLElement("command")}}、{{HTMLElement("link")}}、{{HTMLElement("meta")}}、{{HTMLElement("noscript")}}、{{HTMLElement("script")}}、{{HTMLElement("style")}} 和 {{HTMLElement("title")}}。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:21]:
```suggestion
属于这一类的元素有：{{HTMLElement("base")}}、{{HTMLElement("command")}} {{Deprecated_Inline}}、{{HTMLElement("link")}}、{{HTMLElement("meta")}}、{{HTMLElement("noscript")}}、{{HTMLElement("script")}}、{{HTMLElement("style")}} 和 {{HTMLElement("title")}}。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 2]
```diff
@@ -2,96 +2,98 @@
...
+流式内（Flow content）容是一个广泛的类别，包括大多数可以包含在 {{HTMLElement("body")}} 元素之内的元素，包括标题元素、分段元素、措辞元素、嵌入元素、互动元素和表单相关元素。它还包括文本节点（但不包括那些只由空白字符组成的节点）。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:25]:
```suggestion
流式内容（Flow content）是一个广泛的类别，包括大多数可以包含在 {{HTMLElement("body")}} 元素之内的元素，包括标题元素、分段元素、措辞元素、嵌入元素、互动元素和表单相关元素。它还包括文本节点（但不包括那些只由空白字符组成的节点）。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 3]
```diff
@@ -2,96 +2,98 @@
...
-- {{HTMLElement("link")}}, 仅限于**[itemprop](/zh-CN/docs/HTML/Global_attributes#attr-itemprop)**属性存在的情形
-- {{HTMLElement("meta")}}, 仅限于**[itemprop](/zh-CN/docs/HTML/Global_attributes#attr-itemprop)**属性存在的情形
-- {{HTMLElement("style")}}, 仅限于 {{htmlattrxref("scoped","style")}}属性存在的情形
+- {{HTMLElement("area")}}，仅限于它作为 {{HTMLElement("map")}} 的子节点的情形
+- {{HTMLElement("link")}}，仅限于 [itemprop](/zh-CN/docs/Web/HTML/Global_attributes/itemprop) 属性存在的情形
+- {{HTMLElement("meta")}}，仅限于 [itemprop](/zh-CN/docs/Web/HTML/Global_attributes/itemprop) 属性存在的情形
+- {{HTMLElement("style")}}，仅限于 `scoped` {{deprecated_inline}} 属性存在的情形
 
-### 章节元素（Sectioning content）
+### 分节内容
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:36]: 考虑统一成“分段”或“分节”：

```suggestion
### 分段内容
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 4]
```diff
@@ -2,96 +2,98 @@
...
 
-隶属于分节内容模型的元素 在当前的大纲中创建一个[分节](/zh-CN/docs/Sections_and_Outlines_of_an_HTML5_document)，此分节将定义{{HTMLElement("header")}}元素、{{HTMLElement("footer")}} 元素和标题元素（[heading content](#Heading_content)）的范围。
+分节内容（Sectioning content）是流式内容的一个子集，可以在任何可以使用流式内容的地方出现。属于分段内容模型的元素创建了一个[当前大纲中的分段](/zh-CN/docs/Web/HTML/Element/Heading_Elements)，它定义了 {{HTMLElement("header")}} 元素、{{HTMLElement("footer")}} 元素和标题内容（[heading content](#标题内容)）的范围。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:38]: 同上方标题：

```suggestion
分段内容（Sectioning content）是流式内容的一个子集，可以在任何可以使用流式内容的地方出现。属于分段内容模型的元素创建了一个[当前大纲中的分段](/zh-CN/docs/Web/HTML/Element/Heading_Elements)，它定义了 {{HTMLElement("header")}} 元素、{{HTMLElement("footer")}} 元素和[标题内容](#标题内容)的范围。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 5]
```diff
@@ -2,96 +2,98 @@
...
 
-属于此类的元素有： {{HTMLElement("article")}}, {{HTMLElement("aside")}}, {{HTMLElement("nav")}} and {{HTMLElement("section")}}.
+属于此类的元素有：{{HTMLElement("article")}}、{{HTMLElement("aside")}}、{{HTMLElement("nav")}} 和 {{HTMLElement("section")}}。
 
-> **备注：** 不要将本内容模型与[sectioning root](/zh-CN/docs/Sections_and_Outlines_of_an_HTML5_document#sectioning_root) category 混淆，[sectioning root](/zh-CN/docs/Sections_and_Outlines_of_an_HTML5_document#sectioning_root) category 的作用是把它的内容与常规的大纲隔离。
+### 标题内容
 
-### 标题元素（Heading content）
+标题内容（Heading content）是流式内容的一个子集，定义了分节的标题，而这个分节可能由一个明确的[分节内容](#sectioning_content)元素直接标记，也可能由标题本身隐式地定义。标题内容可以在任何可以使用流式内容的地方出现。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:44]: 同样地，分段/分节

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 6]
```diff
@@ -2,96 +2,98 @@
...
 
-标题内容 定义了分节的标题，而这个分节可能由一个明确的分节内容元素直接标记，也可能由标题本身隐式地定义。
+属于此分类的元素有：{{HTMLElement("h1")}}、{{HTMLElement("h2")}}、{{HTMLElement("h3")}}、{{HTMLElement("h4")}}、{{HTMLElement("h5")}}、{{HTMLElement("h6")}} 和 {{HTMLElement("hgroup")}}。
 
-属于此分类的元素有： {{HTMLElement("h1")}}, {{HTMLElement("h2")}}, {{HTMLElement("h3")}}, {{HTMLElement("h4")}}, {{HTMLElement("h5")}}, {{HTMLElement("h6")}} and {{HTMLElement("hgroup")}}.
+> **备注：** 尽管{{HTMLElement("header")}} 可能包含一些标题内容，但 {{HTMLElement("header")}} 并不是标题内容本身。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:48]:
```suggestion
> **备注：** 尽管 {{HTMLElement("header")}} 可能包含一些标题内容，但其并不是标题内容本身。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 7]
```diff
@@ -2,96 +2,98 @@
...
 
-> **备注：** 尽管{{HTMLElement("header")}}可能包含一些标题内容，但{{HTMLElement("header")}}并不是标题内容本身。
+> **备注：** {{HTMLElement("hgroup")}} 元素已不再推荐使用，因为它不能与辅助性技术正常配合工作。在 HTML 5定稿之前，它已从 W3C 的 HTML 规范中删除，但仍是 WHATWG 规范的一部分，并且至少被大多数浏览器部分支持。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:50]:
```suggestion
> **备注：** {{HTMLElement("hgroup")}} 元素已不再推荐使用，因为它不能与辅助性技术正常配合工作。在 HTML 5 定稿之前，它已从 W3C 的 HTML 规范中删除，但仍是 WHATWG 规范的一部分，并且至少被大多数浏览器部分支持。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 8]
```diff
@@ -2,96 +2,98 @@
...
+属于此类的元素有：{{HTMLElement("abbr")}}、{{HTMLElement("audio")}}、{{HTMLElement("b")}}、{{HTMLElement("bdo")}}、{{HTMLElement("br")}}、{{HTMLElement("button")}}、{{HTMLElement("canvas")}}、{{HTMLElement("cite")}}、{{HTMLElement("code")}}、{{Deprecated_Inline}}{{HTMLElement("command")}}、{{HTMLElement("datalist")}}、{{HTMLElement("dfn")}}、{{HTMLElement("em")}}、{{HTMLElement("embed")}}、{{HTMLElement("i")}}、{{HTMLElement("iframe")}}、{{HTMLElement("img")}}、{{HTMLElement("input")}}、{{HTMLElement("kbd")}}、{{HTMLElement("keygen")}} {{deprecated_inline()}}、{{HTMLElement("label")}}、{{HTMLElement("mark")}}、{{MathMLElement("math")}}、{{HTMLElement("meter")}}、{{HTMLElement("noscript")}}、{{HTMLElement("object")}}、{{HTMLElement("output")}}、{{HTMLElement("progress")}}、{{HTMLElement("q")}}、{{HTMLElement("ruby")}}、{{HTMLElement("samp")}}、{{HTMLElement("script")}}、{{HTMLElement("select")}}、{{HTMLElement("small")}}、{{HTMLElement("span")}}、{{HTMLElement("strong")}}、{{HTMLElement("sub")}}、{{HTMLElement("sup")}}、{{SVGElement("svg")}}、{{HTMLElement("textarea")}}、{{HTMLElement("time")}}、{{HTMLElement("var")}}、{{HTMLElement("video")}}、{{HTMLElement("wbr")}} 和纯文本（仅当所包含的内容不完全为空字符）。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:56]:
```suggestion
属于此类的元素有：{{HTMLElement("abbr")}}、{{HTMLElement("audio")}}、{{HTMLElement("b")}}、{{HTMLElement("bdo")}}、{{HTMLElement("br")}}、{{HTMLElement("button")}}、{{HTMLElement("canvas")}}、{{HTMLElement("cite")}}、{{HTMLElement("code")}}、{{HTMLElement("command")}} {{Deprecated_Inline}}、{{HTMLElement("datalist")}}、{{HTMLElement("dfn")}}、{{HTMLElement("em")}}、{{HTMLElement("embed")}}、{{HTMLElement("i")}}、{{HTMLElement("iframe")}}、{{HTMLElement("img")}}、{{HTMLElement("input")}}、{{HTMLElement("kbd")}}、{{HTMLElement("keygen")}} {{deprecated_inline()}}、{{HTMLElement("label")}}、{{HTMLElement("mark")}}、{{MathMLElement("math")}}、{{HTMLElement("meter")}}、{{HTMLElement("noscript")}}、{{HTMLElement("object")}}、{{HTMLElement("output")}}、{{HTMLElement("progress")}}、{{HTMLElement("q")}}、{{HTMLElement("ruby")}}、{{HTMLElement("samp")}}、{{HTMLElement("script")}}、{{HTMLElement("select")}}、{{HTMLElement("small")}}、{{HTMLElement("span")}}、{{HTMLElement("strong")}}、{{HTMLElement("sub")}}、{{HTMLElement("sup")}}、{{SVGElement("svg")}}、{{HTMLElement("textarea")}}、{{HTMLElement("time")}}、{{HTMLElement("var")}}、{{HTMLElement("video")}}、{{HTMLElement("wbr")}} 和纯文本（仅当所包含的内容不完全为空字符）。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 9]
```diff
@@ -2,96 +2,98 @@
...
-- {{HTMLElement("ins")}}, 当它仅包含 phrasing content 时
-- {{HTMLElement("link")}}, 如果**[itemprop](/zh-CN/docs/HTML/Global_attributes#attr-itemprop)**属性存在的情形
-- {{HTMLElement("map")}}, 当它仅包含 phrasing content 时
-- {{HTMLElement("meta")}},如果**[itemprop](/zh-CN/docs/HTML/Global_attributes#attr-itemprop)**属性存在的情形
+- {{HTMLElement("a")}}，当它仅包含 phrasing content 时
+- {{HTMLElement("area")}}，当它为 {{HTMLElement("map")}} 元素的子元素时
+- {{HTMLElement("del")}}，当它仅包含 phrasing content 时
+- {{HTMLElement("ins")}}，当它仅包含 phrasing content 时
+- {{HTMLElement("link")}}，仅限于 [itemprop](/zh-CN/docs/Web/HTML/Global_attributes/itemprop) 属性存在的情形
+- {{HTMLElement("map")}}，当它仅包含 phrasing content 时
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:65]:
```suggestion
- {{HTMLElement("a")}}，当它仅包含短语内容时
- {{HTMLElement("area")}}，当它为 {{HTMLElement("map")}} 元素的子元素时
- {{HTMLElement("del")}}，当它仅包含短语内容时
- {{HTMLElement("ins")}}，当它仅包含短语内容时
- {{HTMLElement("link")}}，仅限于 [itemprop](/zh-CN/docs/Web/HTML/Global_attributes/itemprop) 属性存在的情形
- {{HTMLElement("map")}}，当它仅包含短语内容时
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 10]
```diff
@@ -2,96 +2,98 @@
...
-- {{HTMLElement("menu")}}, 若{{htmlattrxref("type", "menu")}}属性处于工具栏（toolbar）状态
-- {{HTMLElement("object")}}, 若{{htmlattrxref("usemap", "object")}}属性存在
-- {{HTMLElement("video")}}, 若{{htmlattrxref("controls", "video")}}属性存在
+当内容既不是空的也不是隐藏的时候，它就是可感知（palpable）的；它是被渲染的内容，是实质性的。模型为流动内容的元素应该至少有一个节点是可感知的。
 
-### 表单相关内容（Form-associated content）
+### 表单相关内容
 
-表单相关的内容 包含 拥有表单父节点（exposed by a **form** attribute）的元素，一个表单父节点可以是{{HTMLElement("form")}}元素，也可以是其 id 在表单属性中被指定了的元素。（Form-associated content comprises elements that have a form owner, exposed by a **form** attribute. A form owner is either the containing {{HTMLElement("form")}} element or the element whose id is specified in the **form** attribute.）
+与表单相关的内容（Form-associated content）是流程内容的一个子集，包括有表单所有者的元素，通过**form**属性暴露出来，可以在预期有流式内容的地方使用。表单所有者要么是包含 {{HTMLElement("form")}} 的元素，要么是在**form**属性中指定其 id 的元素。
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:91]:
```suggestion
与表单相关的内容（Form-associated content）是流程内容的一个子集，包括有表单所有者的元素，通过 **form** 属性暴露出来，可以在预期有流式内容的地方使用。表单所有者要么是包含 {{HTMLElement("form")}} 的元素，要么是在 **form** 属性中指定其 id 的元素。
```

File: files/zh-cn/web/guide/html/content_categories/index.md [hunk 11]
```diff
@@ -2,96 +2,98 @@
...
 
 - 主内容类，描述了很多元素共享的内容规范；
 - 表单相关的内容类，描述了表单相关元素共有的内容规范；
 - 特殊内容类，描述了仅仅在某些特殊元素上才需要遵守的内容规范，通常这些元素都有特殊的上下文关系。
 
-[![Content_categories_venn.png](/@api/deki/files/6244/=Content_categories_venn.png?size=webview)](/@api/deki/files/6244/=Content_categories_venn.png)
+[![A Venn diagram showing how the various content categories interrelate. The following sections explain these relationships in text.](content_categories_venn.png)](/en-US/docs/Web/Guide/HTML/Content_categories/content_categories_venn.png)
 
-## 主内容类（Main content categories）
+## 主内容类
```
Comment (yin1999) [inline files/zh-cn/web/guide/html/content_categories/index.md:15]:
```suggestion
## 主内容分类
```

**Review by yin1999 — COMMENTED** [review]
Thanks, I have some suggestions about this.

File: files/zh-cn/web/guide/html/content_categories/index.md (hunk 5 above)
Comment (jasonren0403) [inline files/zh-cn/web/guide/html/content_categories/index.md:44]:
```suggestion
标题内容（Heading content）是流式内容的一个子集，定义了分段的标题，而这个分段可能由一个明确的[分段内容](#分段内容)元素直接标记，也可能由标题本身隐式地定义。标题内容可以在任何可以使用流式内容的地方出现。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**